src/App.jsx

```diff
@@ -20,6 +20,7 @@ const App = () => {
   };
 
   // get the actor list initially on render
+  // TODO: switch to useQuery to run simultaneously to the render instead of after initial render
```

Comment:
I'm curious, why would running simultaneously with the render make this better than running after the render? Shouldn't it happen before the render so that the render has this data available or am I missunderstanding?

Reply:
As far as I understand, it's able to run "simultaneously" and still populate with the data on that first render.

Reply:
Interesting, I'll have to read about this at some point